modules/mod_base/controllers/controller_postback.erl

```diff
@@ -28,6 +28,7 @@
 
 -include_lib("zotonic.hrl").
 
+-define(SIDEJOB_TIMEOUT, 1000).
```

Comment:
The comet flush is now set to: 

```
-define(COMET_FLUSH_DATA,  100).
```

So the `SIDEJOB_TIMEOUT` must be something like 90 msec.